plutus-core/untyped-plutus-core/testlib/Evaluation/Builtins/Definition.hs

```diff
@@ -1251,6 +1255,11 @@ test_definition =
         , test_HashSizes
         , test_SignatureVerification
         , test_BLS12_381
+        , test_integer_ring_properties
+        , test_integer_div_mod_properties
+        , test_integer_quot_rem_properties
+        , test_integer_order_properties
+        , test_integer_ring_properties
```

Comment:
makes sense to rename the test below this one for its name to be consistent:
`test_expModInteger_properties` -> `test_integer_exp_mod_properties`

Reply:
Good idea.